main_app/templates/main_app/women_laws.html

```diff
@@ -97,16 +106,13 @@ <h1 class="wb-stl-heading1">
                     <h5>The Commission of Sati (prevention) Act(1987)</h5>
                 </div>
                 <div class="collapsible-body">
-                    <p>Sati (Prevention) Act, 1987 is a law enacted by Government of Rajasthan in 1987.
+                    <p class="description"><span style="background:#a8dadc">Sati (Prevention) Act</span>, 1987 is a law enacted by Government of Rajasthan in 1987.
```

Comment:
@AishwaryaKaranth Please edit the content in bullet form for every law. Right now, it is less bit unreadable.